code/src/cljs/sixsq/nuvla/ui/edges/utils.cljs

```diff
@@ -153,18 +153,27 @@
                   full-text-search
                   (when extra extra))})
 
+(defn remove-vpn-service
+  [file]
+  (str/replace file #"\n\n  vpn-client:(\n\s{4,}.*)*\n" "\n  "))
```

Comment:
@schaubl @MTrost Fragile regex to suppress a vpn-client from docker-compose in UI code. 

Isn't it cleaner to do like for peripheral and put the vpn-client in a separate docker-compose file in nuvlaedge/deployment repo. Of course this will only be done for recent release of NuvlaEdge but this is not a problem I think

Reply:
Yes, that was Konstan's feedback too above.

Currently on hold anyway until discussion in the tasklist is resolved.

Reply:
@0xbase12 
Sorry, I forgot to remove you here as assignee because it's currently on hold I think